boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -314,7 +328,7 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
             if (EndAddress <= RealBaseAddress)
             {
                 /* This doesn't span any page, so continue with next range */
-                TRACE("Skipping aligned range < PAGE_SIZE. (PcBiosMapCount = %lu, BaseAddress = %lu, Length = %lu)\n",
+                TRACE("Skipping aligned range < PAGE_SIZE. (would-be-PcBiosMapCount = %lu, BaseAddress = %lu, Length = %lu)\n",
```

Comment:
Why is this would-be? According to the argument, it is exactly PcBiosMapCount.

Reply:
It currently is, indeed.
I am using "would-be" as hints that PcBiosMapCount will not be incremented, so current entry is/was valid for current loop iteration only.
You may suggest a better wording.